app/org/sagebionetworks/bridge/models/schedules/ScheduleContext.java

```diff
@@ -176,12 +175,14 @@ public Builder withUserDataGroups(Set<String> userDataGroups) {
             contextBuilder.withUserDataGroups(userDataGroups);
             return this;
         }
-        public Builder withNow(DateTime now) {
-            this.now = now;
+        public Builder withStartsOn(DateTime startsOn) {
+            this.startsOn = startsOn;
             return this;
         }
         public Builder withAccountCreatedOn(DateTime accountCreatedOn) {
-            this.accountCreatedOn = new DateTime(accountCreatedOn, DateTimeZone.UTC);
+            if (accountCreatedOn != null) {
+                this.accountCreatedOn = new DateTime(accountCreatedOn, DateTimeZone.UTC);    
+            }
```

Comment:
This change has uncovered 2 more test gaps. We don't have tests for:

* withContext() doesn't test accountCreatedOn. (I mean, we found out indirectly that it does, but if withContext() suddenly stopped copying accountCreatedOn, we'd have no way of detecting that.)
* withAccountCreatedOn() null case.